src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -233,6 +235,8 @@ define(function (require, exports, module) {
                 // only if it's not an HTML file
                 if(!Content.isHTML(Path.extname(path))) {
                     Handlers.handleFile(path, data);
```

Comment:
The [handleFile] (https://github.com/humphd/brackets/blob/bramble/src/filesystem/impls/filer/lib/handlers.js#L27) function calls the  [createURL] (https://github.com/humphd/brackets/blob/bramble/src/filesystem/impls/filer/BlobUtils.js#L84) function which calls the cache function and ultimately caches the CSS file. This was changed in the [last commit] (https://github.com/humphd/brackets/pull/181). Removing this line would mean that the CSS file wouldn't be cached, the PostMessageTransport wouldn't be able to replace the filename with the Blob URL, and the live preview wouldn't show any CSS.